crates/containerd-shim-wasmedge/Cargo.toml

```diff
@@ -8,7 +8,8 @@ containerd-shim = { workspace = true }
 containerd-shim-wasm = { workspace = true }
 log = { workspace = true }
 ttrpc = { workspace = true }
-wasmedge-sdk = { version = "0.10.1", features = [ "standalone", "static" ] }
+wasmedge-sdk = { version = "0.11.0" }
```

Comment:
`0.11.2` is already available, can we use it?

Reply:
released a few hours ago :-)